### PR TITLE
Add fallbackToGlobal to telemetry

### PR DESCRIPTION
**Pull request by neha-bhargava**

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs [hunk 1]
```diff
@@ -106,7 +106,7 @@ public string GetLastRequestHeader()
 
         /// <summary>
         /// Expected format: 2|api_id,force_refresh|platform_config
-        /// platform_config: region,region_source,is_token_cache_serialized,user_provided_region,validate_use_region,is_legacy_cache_enabled
+        /// platform_config: region,region_source,is_token_cache_serialized,user_provided_region,validate_use_region,is_legacy_cache_enabled,fallback_to_global
```
Comment (jennyf19) [inline src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs:109]: we haven't shipped yet, do we want to put all the regional stuff together? so switch it w/the legacy cache?

**Review by pmaytak — APPROVED** [review]

File: src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs (hunk 1 above)
Comment (pmaytak) [reply src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs:109]: btw would have to move `is_token_cache_serialized` too, it's in the middle of regional as well (unless that was already released previously?)

Comment (neha-bhargava) [reply src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs:109]: `is_token_cache_serialized` was released previously. I too had a thought if I should move it together. Let me make that change real quick -- switch is_legacy_cache_enabled with fallback_to_global

**Review by trwalke — APPROVED** [review]
Thanks Neha. this looks good

**Pull request merged**